### ISSUE TITLE
Expect in a setup test not allowed. Expect must be inside of a test block

### ISSUE THREAD

**Issue by Ghislain89** [playwright-community/eslint-plugin-playwright#290]
Please consider the code provided below. A setup project performs a deployment prior to executing the test. Just like described in the Playwright docs, [here](https://playwright.dev/docs/test-global-setup-teardown#setup). 

It's extremely beneficial to be able to expect that certain conditions have been met before actually starting the tests.  Unfortunately, in version 1.6.0 of this plugin, it is no longer allowed to add an expect to a test, used as a setup.

This is very counterintuitive and I'd argue that below should be an allowed approach.

```TS
import { expect, test as setup } from '@playwright/test';

setup('Perform some setup', async () => {
  await setup.step('Perform the first step', async () => {
    const deploymentStatus = await DeploySomeThingies();
    expect(deploymentStatus).toEqual('success'); 
  });
// ... other steps omitted for brevity
});
```

**Comment by mskelton** [comment]
@Ghislain89 You can add `setup` as a global alias:

https://github.com/playwright-community/eslint-plugin-playwright#aliased-playwright-globals

**Comment by Ghislain89** [comment]
That makes sense, I overlooked this in the docs. Thanks!
